DataFormats/METReco/src/classes_def.xml

```diff
@@ -164,11 +164,15 @@
   </class>
   <class name="edm::Wrapper<reco::EcalHaloData>"/>
```

Comment:
Looks like ```HaloTowerStrip``` doesn't have a statement that it should have its dictionary generated. There is only one for ```std::vector<HaloTowerStrip>```